dolphinscheduler-ui/src/views/resource/file/upload/index.tsx

```diff
@@ -38,15 +38,16 @@ export default defineComponent({
     const { handleUploadFile } = useUpload(state)
 
     const hideModal = () => {
+      state.uploadForm.name = ''
```

Comment:
The `resetForm ` function above can reset the form. line 37

Reply:
😥 got it!